test/validation/org/apache/cassandra/HarnessTester.java

```diff
@@ -24,10 +24,12 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.lang.Throwable;
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.Arrays;
```

Comment:
I tried putting file names directly into Collection<Object[]> yesterday but that wasn't working. I tried it now and seems to be working. I might be missing something yesterday.